Documentation/edgefs-csi.md

```diff
@@ -364,5 +364,5 @@ Please submit an issue at: [Issues](https://github.com/Nexenta/edgefs-csi/issues
 - Show termination message in case driver failed to run:
 
   ```bash
-  kubectl get pod edgefs-iscsi-csi-controller-0 -o go-template="{{range .status.containerStatuses}}{{.lastState.terminated.message}}{{end}}"
+  kubectl get pod edgefs-iscsi-csi-controller-0 -o go-template="{% raw %}{{range .status.containerStatuses}}{{.lastState.terminated.message}}{{end}}{% endraw %}"
```

Comment:
from a little quick experimentation, it looks like these new raw tags will be included in the output of this command, so when a user copies and pastes it from the markdown on github they will see this output.  Example:
```
> kubectl get pod mypod -o go-template="{% raw %}{{range .status.conditions}}{{.type}} {{end}}{% endraw %}"
{% raw %}Initialized Ready ContainersReady PodScheduled {% endraw %}
```

Note this would not be a problem on the rook.io docs site itself if the static generator strips out these tags, but we have to consider users that consume the original markdown docs as well and make sure the output is friendly for them too.

Is there a way to do this that is rook.io docs site friendly and raw markdown friendly too?

Reply:
we could certainly wrap the entire triple tick bash section with the raw tags instead. Would that achieve what you are looking for?

Reply:
that sounds reasonable @rathpc, please update this PR with that change and we'll see how it looks :)

Reply:
@jbw976 The PR has been updated